### DISCUSSION TITLE
Bleachbit 5.0 and linuxmint 21.1 Mate

### DISCUSSION THREAD

**Discussion by christg29** [bleachbit/bleachbit#1565] [General]
Hello everyone,
I have downloaded Bleachbit 5.0 for Ubuntu following this link:
https://download.bleachbit.org/beta/4.5.0/bleachbit_4.5.0-0_all_ubuntu2204.deb
The installation went well.
However, the application won't launch.
is this a bug? 
here's what I can find in syslog :

AptDaemon: INFO: InstallFile() was called: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:19 ZZZZZZ org.debian.apt[3645]: 10:28:19 AptDaemon [INFO]: InstallFile() was called: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:19 ZZZZZZ AptDaemon.Trans: INFO: Simulate was called
Oct 21 10:28:19 ZZZZZZ org.debian.apt[3645]: 10:28:19 AptDaemon.Trans [INFO]: Simulate was called
Oct 21 10:28:20 ZZZZZZ systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Oct 21 10:28:26 ZZZZZZ AptDaemon.Worker: INFO: Simulating trans: /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:26 ZZZZZZ org.debian.apt[3645]: 10:28:26 AptDaemon.Worker [INFO]: Simulating trans: /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:26 ZZZZZZ AptDaemon.Worker: WARNING: An additional step to open the cache is required
Oct 21 10:28:26 ZZZZZZ org.debian.apt[3645]: 10:28:26 AptDaemon.Worker [WARNING]: An additional step to open the cache is required
Oct 21 10:28:26 ZZZZZZ AptDaemon.Worker: INFO: Installing local package file: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:26 ZZZZZZ org.debian.apt[3645]: 10:28:26 AptDaemon.Worker [INFO]: Installing local package file: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:26 ZZZZZZ AptDaemon.Worker: WARNING: An additional step to open the cache is required
Oct 21 10:28:26 ZZZZZZ org.debian.apt[3645]: 10:28:26 AptDaemon.Worker [WARNING]: An additional step to open the cache is required
Oct 21 10:28:27 ZZZZZZ AptDaemon.Trans: INFO: Queuing transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:27 ZZZZZZ org.debian.apt[3645]: 10:28:27 AptDaemon.Trans [INFO]: Queuing transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:27 ZZZZZZ AptDaemon.Worker: INFO: Processing transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:27 ZZZZZZ org.debian.apt[3645]: 10:28:27 AptDaemon.Worker [INFO]: Processing transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:27 ZZZZZZ AptDaemon.Worker: WARNING: An additional step to open the cache is required
Oct 21 10:28:27 ZZZZZZ org.debian.apt[3645]: 10:28:27 AptDaemon.Worker [WARNING]: An additional step to open the cache is required
Oct 21 10:28:27 ZZZZZZ AptDaemon.Worker: INFO: Installing local package file: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:27 ZZZZZZ org.debian.apt[3645]: 10:28:27 AptDaemon.Worker [INFO]: Installing local package file: /home/vxxxxx/Téléchargements/bleachbit_4.5.0-0_all_ubuntu2204.deb
Oct 21 10:28:27 ZZZZZZAptDaemon.Worker: WARNING: An additional step to open the cache is required
Oct 21 10:28:27 ZZZZZZ org.debian.apt[3645]: 10:28:27 AptDaemon.Worker [WARNING]: An additional step to open the cache is required
Oct 21 10:28:29 ZZZZZZ org.debian.apt[3645]: /usr/lib/python3/dist-packages/aptdaemon/progress.py:490: Warning: Source ID 31 was not found when attempting to remove it
Oct 21 10:28:29 ZZZZZZ org.debian.apt[3645]:   GLib.source_remove(id)
Oct 21 10:28:29 ZZZZZZ org.debian.apt[3645]: /usr/lib/python3/dist-packages/aptdaemon/progress.py:490: Warning: Source ID 32 was not found when attempting to remove it
Oct 21 10:28:29 ZZZZZZ org.debian.apt[3645]:   GLib.source_remove(id)
Oct 21 10:28:29 ZZZZZZ AptDaemon.Worker: INFO: Finished transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:29 ZZZZZZ org.debian.apt[3645]: 10:28:29 AptDaemon.Worker [INFO]: Finished transaction /org/debian/apt/transaction/5ddba5a3e1d94d27a982f5bd7cc681ed
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]: Traceback (most recent call last):
Oct 21 10:28:48 EQUINOXE org.mate.panel.applet.MintMenuAppletFactory[3747]:   File "/usr/bin/bleachbit", line 28, in <module>
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]:     from bleachbit import _
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]: ModuleNotFoundError: No module named 'bleachbit'
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]: Error in sys.excepthook:
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]: Traceback (most recent call last):
Oct 21 10:28:48 ZZZZZZ org.mate.panel.applet.MintMenuAppletFactory[3747]:   File "/usr/lib/python3/dist-packages/apport_python_hook.py", l

**Comment by rados** [comment]
@christg29 You might want to try this solution: https://github.com/bleachbit/bleachbit/issues/1550#issuecomment-1768216420

**Reply by christg29** [reply]
Hello,
Thanks for your reply
I have removed version 5.0 and installed Bleachit version 4.4.2. 
This version 4.4.2 runs correctly. i can use it
but I'll say this
When using Bleachbit for Browser FireFox Version 118.0.2 and Brave [Version 1.59.120]:
FireFox - Bleachbit does the job and reports with no errors.
Brave -    BleachBit does the job but reports an error:

Nettoyer le fichier 0B /home/xxxxxx/.config/chromium/Default/History
Nettoyer le fichier 0B /home/xxxxxx/.config/chromium/Default/Favicons
Erreur : chromium.form_history : Function: Nettoyer le fichier: /home/xxxxxx/.config/chromium/Default/Web Data
Traceback (most recent call last):
  File "/usr/share/bleachbit/bleachbit/FileUtilities.py", line 474, in execute_sqlite3
    cursor.execute(cmd)
sqlite3.OperationalError: no such table: autofill_profile_names
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/share/bleachbit/bleachbit/Command.py", line 160, in execute
    self.func(self.path)
  File "/usr/share/bleachbit/bleachbit/Special.py", line 106, in delete_chrome_autofill
    FileUtilities.execute_sqlite3(path, cmds)
  File "/usr/share/bleachbit/bleachbit/FileUtilities.py", line 481, in execute_sqlite3
    raise sqlite3.OperationalError(
sqlite3.OperationalError: no such table: autofill_profile_names: /home/xxxxxx/.config/chromium/Default/Web Data
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/share/bleachbit/bleachbit/Worker.py", line 87, in execute
    for ret in cmd.execute(self.really_delete):
  File "/usr/share/bleachbit/bleachbit/Command.py", line 162, in execute
     if -1 == e.message.find('file is encrypted or is not a database') and \
AttributeError: 'OperationalError' object has no attribute 'message'
Nettoyer le fichier 0B /home/xxxxxx/.config/chromium/Default/Web Data

Thank you
